src/clj/rems/api/schema.clj

```diff
@@ -143,6 +144,7 @@
    :license/start DateTime
    :license/end (s/maybe DateTime)
    :license/enabled s/Bool
+   :license/active s/Bool
```

Comment:
this missing was caught only by our browser tests. I'll follow up with another PR that uses the v2 application api more widely in tests